analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -32,9 +32,9 @@ import org.ossreviewtoolkit.model.AnalyzerResult
 import org.ossreviewtoolkit.model.AnalyzerRun
 import org.ossreviewtoolkit.model.OrtResult
 import org.ossreviewtoolkit.model.Repository
-import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
 import org.ossreviewtoolkit.model.config.RepositoryConfiguration
+import org.ossreviewtoolkit.model.orEmpty
```

Comment:
Looks like you missed to replace this:

https://github.com/oss-review-toolkit/ort/blob/a2d82888eecc3049e1872140b20359e486207d61/analyzer/src/main/kotlin/managers/Npm.kt#L234